### PR TITLE
create test cases for  SwaggerConfiguration(#508)

### PR DESCRIPTION
**Pull request by Lin1nGithub**
Make sure that:

- [x] You have read the [contribution guidelines](https://dromara.org/en-us/docs/soul/contributor.html).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [ ] Your local test passed `mvn clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import org.mockito.junit.MockitoJUnitRunner;
+
+
+/**
+ * Test cases for SwaggerConfiguration.
+ *
+ * @author linkuan
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class SwaggerConfigurationTest {
```
[the remark this answers is not in the document]
Comment (Lin1nGithub) [reply soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java:33]: what means final?

**Comment by songyuequan** [comment]
too many  commits

File: soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (songyuequan) [reply soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java:33]: make the class final

[the remark this answers is not in the document]
Comment (Lin1nGithub) [reply soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java:33]: ok,I got it~

File: soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+package org.dromara.soul.admin.config;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.MockitoJUnitRunner;
+
+
```
Comment (songyuequan) [inline soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java:26]: remove empty line

Comment (Lin1nGithub) [reply soul-admin/src/test/java/org/dromara/soul/admin/config/SwaggerConfigurationTest.java]: have removed

**Comment by Lin1nGithub** [comment]
reset commit

**Pull request closed without merging**